packages/mdc-textfield/mdc-textfield.scss

```diff
@@ -423,6 +488,7 @@ $mdc-textfield-disabled-border-on-dark: rgba(white, .3);
   // Account for offset by border
   .mdc-textfield__input {
     padding-bottom: 7px;
```

Comment:
I think this can be removed now (it actually causes an offset rather than preventing one)